packages/utils-helpers/src/urlWithParams.ts

```diff
@@ -3,7 +3,11 @@
 //
 export function urlWithParams(urlString: string, params: any = {}): string {
     const queryString = encodeAsQueryString(params);
-    return urlString + (queryString === '' ? '' : '?' + queryString);
+    if (queryString === '') {
+        return urlString;
+    }
+    const url = new URL(urlString);
+    return urlString + (url.search === '' ? '?' : '&') + queryString;
```

Comment:
Note: This is just a very basic extension. It won't merge all params.
i.e. `urlWithParams('http://neos.io?foo=bar', {foo: 'baz'})` would return `http://neos.io?foo=bar&foo=baz`.

If anyone feels like a proper implementation that supports PHP-compatible query strings and merges the params recursively.. feel free :)